Services/Repository/GlobalScreen/classes/RepositoryMainBarProvider.php

```diff
@@ -118,33 +118,31 @@ public function getStaticSubItems(): array
 
         $title = $this->dic->language()->txt("mm_favorites");
         $icon = $this->dic->ui()->factory()->symbol()->icon()->custom(ilUtil::getImagePath("icon_fav.svg"), $title);
-        $entries[] = $this->mainmenu->complex($this->if->identifier('mm_pd_sel_items'))
-                       ->withSupportsAsynchronousLoading(true)
-                       ->withTitle($title)
-                       ->withSymbol($icon)
-                       ->withContentWrapper(function () use ($f): Legacy {
-                           $fav_list = new \ilFavouritesListGUI();
-
-                           return $f->legacy($fav_list->render());
-                       })
-                       ->withParent(StandardTopItemsProvider::getInstance()->getPersonalWorkspaceIdentification())
-                       ->withPosition(10)
-                       ->withAvailableCallable(
-                           function () use ($dic): bool {
-                               return (bool) $dic->settings()->get('rep_favourites', "0");
-                           }
-                       )
-                       ->withVisibilityCallable(
-                           $access_helper->isUserLoggedIn($access_helper->isRepositoryReadable(
-                               static function () use ($dic): bool {
-                                   return true;
-                                   $pdItemsViewSettings = new ilPDSelectedItemsBlockViewSettings($dic->user());
-
-                                   return $pdItemsViewSettings->allViewsEnabled() || $pdItemsViewSettings->enabledSelectedItems();
-                               }
-                           ))
-                       );
 
+        $entries[] = $this->mainmenu->complex($this->if->identifier('mm_pd_sel_items'))
+                                    ->withSupportsAsynchronousLoading(true)
+                                    ->withTitle($title)
+                                    ->withSymbol($icon)
+                                    ->withContentWrapper(function () use ($f): Legacy {
+                                        return $f->legacy((new \ilFavouritesListGUI())->render());
+                                    })
+                                    ->withParent(StandardTopItemsProvider::getInstance()->getPersonalWorkspaceIdentification())
+                                    ->withPosition(10)
+                                    ->withAvailableCallable(
+                                        function () use ($dic): bool {
```

Comment:
Please add `static function` and consider using `static fn` (if it can be formatted nicely).